### PR TITLE
added contributing.md

### PR DESCRIPTION
**Pull request by engelmi**
Adding a CONTRIBUTING.md based on the generator from Eclipse: https://www.eclipse.org/projects/tools/documentation.php

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+that each of your contributions will comply with the commitments documented in
+the Developer Certificate of Origin (DCO). Having an ECA on file associated with
+the email address matching the "Author" field of your contribution's Git commits
+fulfills the DCO's requirement that you sign-off on your contributions.
+
+For more information, please see the [Eclipse Committer Handbook](https://www.eclipse.org/projects/handbook/#resources-commit)
+
+## Contact
+
+Contact the project developers via the project's "dev" list [here](https://accounts.eclipse.org/mailing-list/bluechi-dev).
```
Comment (mwperina) [inline CONTRIBUTING.md]: "project's "dev" list" -> "project's development mailing list"

File: CONTRIBUTING.md
Comment (mwperina) [inline CONTRIBUTING.md:1]: We should probably update https://github.com/eclipse-bluechi/bluechi#how-to-contribute (merge information) and have all contribution description here

File: CONTRIBUTING.md (hunk 1 above)
Comment (engelmi) [reply CONTRIBUTING.md]: This was from the eclipse generator, but I agree. Changing.

File: CONTRIBUTING.md
Comment (engelmi) [reply CONTRIBUTING.md:1]: Oh, I missed that - good point! I'll move the content, link it in the README and adjust here.

Comment (engelmi) [reply CONTRIBUTING.md:1]: Done. 
PATL @mwperina

**Review by mwperina — APPROVED** [review]
LGTM

File: CONTRIBUTING.md
Comment (mwperina) [reply CONTRIBUTING.md:1]: :+1:

**Pull request merged**